blocks/subscriptions-block/intl.json

```diff
@@ -6852,9 +6852,39 @@
 	"checkout-block.account": {
 		"en": "Account"
 	},
+	"checkout-block.address1": {
+		"en": "Address 1"
+	},
+	"checkout-block.address1-validation": {
+		"en": "Please enter a valid address."
+	},
+	"checkout-block.address2": {
+		"en": "Address 2 (Optional)"
+	},
+	"checkout-block.address2-placeholder": {
+		"en": "ex: Apt/Suite/Floor"
+	},
 	"checkout-block.billingAddress": {
 		"en": "Billing Address"
```

Comment:
I'm checking with pardis if we should keep all these strings in lowercase except the first one (i.e. Billing address), I cannot see a pattern on the designs about the use of uppercase & lowercase.